workshops/portfolio/README.md

```diff
@@ -0,0 +1,563 @@
+
+![](img/css_curly_braces.png) I want all the style properties inside the curly brackets to apply:
+
+![](img/css_property.png) specifically, I want the width
+
+![](img/css_value.png) to be 50% of the width of the page.
+
+**Vocabulary**
+
+![](img/img_css_highlight.png) `img` is called the selector, it "selects" all
+of the `img` tags and applies all of the settings inside of the curly braces
+
+![](img/css_curly_braces.png) `{` `}` are called curly braces
+
+![](img/css_property.png) the thing on the left side of the semicolon
+is called the `"property", in this case it is `width`
+
+![](img/css_value.png) `50%` is what's known as the `"value"`
+
+### But what happens if I resize the page?
+
+![](img/resize_page.gif)
+
+
+Hmm... I don't want my image to change sizes if my page changes sizes.
+
+## Pixels
+
+Lets change the units from a percentage to pixels.
+This will now set the width of the image to always be `200px` or 200 pixels
+no matter the size of the page.
+
+![](img/pixels.gif)
+
+And if we try to resize the page:
+
+![](img/nothing_happens.gif)
+
+nothing happens.
+
+## We've made good progress:
+
+This is where we want to go:
+
+> ![](img/final_1.png)
+
+We're here right now:
+
+![](img/current.png)
+
+## Making the image a circle
+
+While we're working on the image, let's make it nice and rounded.
+
+We're going to use one of the most useful tools in programming to figure out how to do this, the great and mighty Google.
+
+To figure out how to make an image nice and rounded, I might  Google something
+like, `make image circular in css`
+
+This is the first result I get
+
+> ![](img/rounded_image.png)
+
+I open it and see this site:
+
+> ![](img/how_to_circular.png)
+
+I focus on this line:
+
+> ![](img/how_to_circular_highlight.png)
+
+It seems like I just need to add
+
+```javascript
+border-radius: 50%;
+```
+
+Since I want this to apply my `img` tag, I add here:
+
+![](img/border_radius.gif)
+
+
+Hey! It works!
+
+```
+Note that this was just a brief introduction to using Google to figure out
+how to do something. You will be doing this **_A LOT_** more of this in the
+future.
+```
+
+## We've made more progress!
+
+This is where we want to go:
+
+> ![](img/final_1.png)
+
+We're here right now:
+
+![](img/current_2.png)
+
+Now we have to:
+
+- center everything
+- change the font
+
+## Adding the body tag
+
+We want to center everything in the entire page.
+
+1. We can add a tag called the `body` tag
+2. put everything inside of the body tag
+3. and now that everything is inside the body tag, if we tell the CSS to center
+   the body, it will center **everything** inside of the body.
+
+### 1. We can add a tag called the `body` tag
+
+![](img/body.gif)
+
+### 2. put everything inside of the body tag
+
+![](img/body_copy.gif)
+
+<!-- ### indent properly
+
+Because the `img`, `h1`, and `p` tag are all inside of the `body` tag now,
+they need to be indented to be more readible:
+
+![](img/indentation.gif) -->
```

Comment:
What if I want to leave a note that this might be something that might be considered in the future?